misc-utils/meson.build

```diff
@@ -76,15 +76,17 @@ test_uuidd_sources = files(
   'test_uuidd.c',
 )
 
-if build_liblastlog2 and systemd.found()
```

Comment:
In this case, we also need a default for $tmpfilesdir in ./configure.ac. Now it depends on `pkg-config systemd`.

Reply:
I've never written autotools config before and I have no idea how to correctly do this... My main intention with this PR is to switch to `StateDirectory=`. Now that it is refused I'm tempted to close this.